### ISSUE TITLE
Collaborating on Conan packages

### ISSUE THREAD

**Issue by solvingj** [barcharcraz/conan-packages#1]
Hi Charlie, 

You have made many good Conan packages and our team would like to collaborate on some of them with you.  Please let me know if you have any interest.

**Comment by Croydon** [comment]
I agree. You have made several interesting Conan recipes and we would like to work with you at https://github.com/bincrafters
